README.md

```diff
@@ -78,4 +78,26 @@ func main() {
 }
 ```
 
-</details>
\ No newline at end of file
+</details>
+
+## Adapters
+
+### Datadog
+
+To ensure that Datadog outputs logs in the correct format Datadog must be
+configured with a custom logger.
+
+```go
+package main
+
+import (
+	datadogLogger "github.com/coopnorge/go-logger/adapter/datadog"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+)
+
+func main() {
+	l, err =: gormLogger.NewLogger(gormLogger.WithGlobalLogger())
+	ddtrace.UseLogger(l)
+}
+```
```

Comment:
I can't use GitHub's suggestion-feature, but:
`gormLogger` -> `datadogLogger`

And: Add a newline at the end of the file (we should also consider bringing in a linter for this)